tests/v2/test_waku_filter.nim

```diff
@@ -17,7 +17,12 @@ import
 procSuite "Waku Filter":
 
   test "encoding and decoding FilterRPC":
-    let rpc = FilterRPC(filters: @[ContentFilter(topics: @["foo", "bar"])])
+    let 
+      peer = PeerInfo.init(PrivateKey.random(ECDSA, rng[]).get())
+      rpc = FilterRPC(
+        filterRequest: @[FilterRequest(contentFilter: @[ContentFilter(topics: @["foo", "bar"])], topic: "foo")],
+        messagePush: @[MessagePush(message: @[Message.init(peer, @[byte 1, 2, 3], "topic", 3, false)])]
```

Comment:
Does this correspond to `  repeated Message message = 1;`? The way I naively read this it looks like there is a single `message` field with Messages in it.

`[message1 message2 ...]` vs `{message: [message1 message2 ...]}`

Maybe I'm misreading it though

Reply:
message: `@[]` means it's a sequence of messages, in the defined its exactly a translation from this:

```protobuf
message MessagePush {
  repeated Message message = 1;
}
```

Reply:
Ah yes, brain fart, we should probably rename protobuf to be `messages` to be more idiomatic